src/__test__/components.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from '../redux/store';
+import Contact from '../Components/Pages/Contact';
+import Details from '../Components/Pages/Details';
+import Home from '../Components/Pages/Home';
+import '@testing-library/jest-dom';
+
+describe('Testing Profile and Mission Components', () => {
+  it('Testing MissionContainer components', () => {
+    const home = render(
+      <React.StrictMode>
+        <Router>
+          <Provider store={store}>
+            <Home />
+          </Provider>
+        </Router>
+      </React.StrictMode>,
+    );
+    expect(home).toMatchSnapshot();
+  });
+
+  it('Testing MissionContainer components', () => {
+    const contact = render(
+      <React.StrictMode>
+        <Router>
+          <Provider store={store}>
+            <Contact />
+          </Provider>
+        </Router>
+      </React.StrictMode>,
+    );
+    expect(contact).toMatchSnapshot();
+  });
+
+  it('Testing MissionContainer components', () => {
+    const details = render(
+      <React.StrictMode>
+        <Router>
+          <Provider store={store}>
+            <Details />
+          </Provider>
+        </Router>
+      </React.StrictMode>,
+    );
+    expect(details).toMatchSnapshot();
+  });
```

Comment:
As requested by the previous reviewer, you also need to write the unit tests for the pure functions in your redux code. Kindly do as and then submit for a re-review.

Reply:
Hi, @muneebulrehman 
Thanks for the review.
I think what I did was also a unit test
I think there are various tools to do unit testing in react.
I used the one I understood best which is the snap shot

Reply:
- [ ] As requested by the previous reviewer, kindly write unit tests as these are snapshot tests which you currently have. Also check your tests as they are currently failing.
![image](https://user-images.githubusercontent.com/81932478/203499886-76f3776f-8b29-4e16-a5b1-8fbdaa9e643a.png)